src/Tools/dotnet-monitor/CollectionRules/Triggers/AspNetResponseStatusTriggerFactory.cs

```diff
@@ -14,10 +19,45 @@ namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Triggers
     internal sealed class AspNetResponseStatusTriggerFactory :
         ICollectionRuleTriggerFactory<AspNetResponseStatusOptions>
     {
+        /// <inheritdoc/>
+        private readonly EventPipeTriggerFactory _eventPipeTriggerFactory;
+        private readonly ITraceEventTriggerFactory<AspNetRequestStatusTriggerSettings> _traceEventTriggerFactory;
+
+        public AspNetResponseStatusTriggerFactory(
+            EventPipeTriggerFactory eventPipeTriggerFactory,
+            ITraceEventTriggerFactory<AspNetRequestStatusTriggerSettings> traceEventTriggerFactory)
+        {
+            _eventPipeTriggerFactory = eventPipeTriggerFactory;
+            _traceEventTriggerFactory = traceEventTriggerFactory;
+        }
+
         /// <inheritdoc/>
         public ICollectionRuleTrigger Create(IEndpointInfo endpointInfo, Action callback, AspNetResponseStatusOptions options)
         {
-            throw new NotImplementedException("TODO: Implement AspNetResponseStatusTrigger.");
+            var settings = new AspNetRequestStatusTriggerSettings
+            {
+                ExcludePaths = options.ExcludePaths,
+                IncludePaths = options.IncludePaths,
+                RequestCount = options.ResponseCount,
+                StatusCodes = options.StatusCodes.Select(ParseRange).ToArray(),
+                SlidingWindowDuration = options.SlidingWindowDuration ?? TimeSpan.Parse(AspNetResponseStatusOptionsDefaults.SlidingWindowDuration, CultureInfo.InvariantCulture),
+            };
+
+            var aspnetTriggerSourceConfiguration = new AspNetTriggerSourceConfiguration();
+
+            return _eventPipeTriggerFactory.Create(endpointInfo, aspnetTriggerSourceConfiguration, _traceEventTriggerFactory, settings, callback);
+        }
+
+        private static StatusCodeRange ParseRange(string range)
```

Comment:
I think this is a good start but I imagine we may want to expand this syntax to support something like: `200,202-204`

Reply:
Each item in the StatusCodes property above represents either a single code or a code range. An individual item should not have a combination of them. So this is permissable:

```json
[ "200", "202-204" ]
```

but not:

```json
[ "200,202-204" ]
```

Reply:
I will leave it as is for now, but if there's strong favor for having the payload be one string I can change it.